src/routes/transactions/__tests__/controllers/list-multisig-transactions-by-safe.transactions.controller.spec.ts

```diff
@@ -140,6 +143,47 @@ describe('List multisig transactions by Safe - Transactions Controller (Unit)',
     const chainId = faker.string.numeric();
     const safeAddress = faker.finance.ethereumAddress();
     const chain = chainBuilder().with('chainId', chainId).build();
+    const safe = safeBuilder()
+      .with('address', '0xC22de4CeeAFd9436d20A9b18C5970D680D3498c8')
```

Comment:
```suggestion
    const safeAddress = '0xC22de4CeeAFd9436d20A9b18C5970D680D3498c8';
    const chain = chainBuilder().with('chainId', chainId).build();
    const safe = safeBuilder()
      .with('address', safeAddress)
```

Reply:
Same as https://github.com/safe-global/safe-client-gateway-nest/pull/503#discussion_r1243921579

Reply:
Changed in [0450df9](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/0450df9f6fa71b7b2f5bd8b364e0fa0aedbdb2f3)